client_end_user/shared/src/commonMain/kotlin/util/ImagePicker.kt

```diff
@@ -0,0 +1,27 @@
+package util
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.ImageBitmap
+
+expect class ImagePickerFactory(context: PlatformContext) {
+
```

Comment:
No need for an image picker here, I don't see any uploading image in the design we can add it later if they need it.

Reply:
Done